src/__test__/redux/actions/componentConfig/__snapshots__/loadPlotConfig.test.js.snap

```diff
@@ -14,7 +14,6 @@ Object {
       "xAxisText": "#UMIs in cell",
       "yAxisText": "#UMIs * #Cells",
     },
-    "binStep": 200,
```

Comment:
These binStep removals are related to this discussion https://this-is-biomage.slack.com/archives/C015CHSUDRU/p1643803335465179